### DISCUSSION TITLE
What Scraper to integrate next?

### DISCUSSION THREAD

**Discussion by itsToggle** [itsToggle/plex_debrid#11] [Polls]
Hey guys,

I dont know if anyone will see this but maybe vote on what scrapers you want to see integrated next.

Jackett is now integrated. It currently only allows jackett sources that return magnet links (or magnet links hidden in sub-pages), so let me know if your favorite tracker works :)

Another option would be Orionoid. This is a paid service that offers a very restricted free version (100 links per day). It is lightning fast however and doesnt rely on up-times of shitty torrent indexer APIs. You do get a lot of junk links though, which is why I have mixed feelings about it.

Thanks!

**Comment by Dexxa1024** [comment]
Radarr and sonarr would be good? :-)   or even ombi

**Reply by itsToggle** [reply]
I don't think Ombi is a scraping service. Isnt that the old method of requesting content for Plex Shares? 

Radarr and Sonarr are both download automation tools. The Scraper that they commonly use is called Jackett, a program that can scrape a huge number of torrent indexers :)

**Comment by beljim** [comment]
Have you heard of Prowlarr ?  Not sure if it's the kind of thing you're thinking of. (https://github.com/Prowlarr/Prowlarr) 
It's what a lot of folk have moved on to after Jackett and Nzbhydra 2.

**Reply by itsToggle** [reply]
Looks awesome! I'll add it as another option alongside jackett. Looks like prowlarr might even be a little faster :)

**Comment by jrjaggari** [comment]
Could you please add https://tamilmv.unblockit.pet/ and https://tamilblasters.unblockit.pet/ to the list of torrent indexers?

**Comment by sasagr** [comment]
Furk.net, maybe it is not a pure scraper but it does also (but not only) what a scraper does. You search BitTorrent sites using their meta search engine, review the information, their servers act as a proxy and download torrents on your behalf. Eventually you get regular HTTP links and download torrents as regular HTTP files. So it does something like Real-Debrid but it has a meta search engine included (so the scraper is included).
I already mentioned this request in the issue section but I will put it here in case other people that are familiar with Furk, want to put a vote to this idea  :)
